create-release.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Creates new package release in git.
+# a skip-merge is about not merge master to release.
+
+set -e
+set -o pipefail
+
+fail() {
+    echo "Error: $*" 1>&2
+    exit 1
+}
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+
+if [[ "$current_branch" != "release" ]]; then
+    echo 'Checkout "release".'
+    git checkout release
+fi
+
+if [[ "$1" == "skip-merge" ]]; then
```

Comment:
I think this "git merge master" is ok for the first time. But if we have some fix in release and master branch at the same time with some conflicts, there will be some problem.

Reply:
Yes, this is why I add a skip-merge to let people choose if they do it manually or not.
I think in your server, you will do with "skip-merge" as there are many remotes there.

Reply:
Also add codes to let ct choose if they want to merge master to release automatically.